File: Go/01.Go-Basic-February-2022/09.advancedLoops-Lab/02.password.go
package main

import "fmt"

func main() {
	var username string
	fmt.Scanln(&username)

	var password string
	fmt.Scanln(&password)

	var enteredPassword string
	fmt.Scanln(&enteredPassword)

	for enteredPassword != password {
		fmt.Scanln(&enteredPassword)
	}

	fmt.Printf("Welcome %s!", username)
}

File: JavaScript/01.JavaScript-Basic-February-2021/09.whileLoop-Lab/02.password.js
function password(input) {
    let username = input[0];
    let password = input[1];

    let index = 2;
    while (input[index] !== password) {
        index++;
    }
    console.log(`Welcome ${username}!`);
}

password(["Nakov", "1234", "Pass", "1324", "1234"]);
password(["Gosho", "secret", "secret"]);